### ISSUE TITLE
Prepare Cohort Call 1 [ 2024-epa ]

### ISSUE THREAD

**Issue by stefaniebutland** [Openscapes/cohort-planning#195]
This is the same for every Cohort Call except the last. Create one issue for every Cohort Call and add the Call number to the issue title.

- [x] create Agenda (kyber); make manual edits
- [x] review roles and timing

**Comment by stefaniebutland** [comment]
Stef will likely be away for this Call. Gayle will manage breakouts as she does for all EPA sessions.

Jeff Hollister will give "Better Science in Less Time" lesson. Current agenda links to his 2023 slide deck. Ping him to update link if needed, once the Agenda is moved to Teams Cohort Folder.

**Agenda status on Mar 28:** Julie and Stef reviewed roles during How We Work for Champions Setup. @jules32 will review the plan for Introductions and Openscapes Mindset in light of 'show not tell'. 
- Mike McManus's 2023 feedback: want opportunity for cohort to meet each other as a whole group at the start because people spent all their time introducing selves in later breakouts.

**Comment by jules32** [comment]
We are all set for tomorrow!

The 01 Call Agenda Word Doc is in  the `Openscapes_CohortCalls [ 2024-epa ] Sharepoint folder, as is the Pathways folder. 

For the Agenda, I did move Jeff's Better Science story to be first just after the Welcome, and the breakout rooms are longer (combined Better Science + ommitted Intro time) to help give people a start to introduce themselves. We've found smaller groups preferable to whole-group intros because of timing, and particularly important for the kick off call. We can continue having longer breakouts and group discussions in the calls to come
